DOC-2419: Math icon state was not set to `active` on math element selection. (#3311)
tinymce/tinymce-docs: modules/ROOT/pages/7.1.2-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== Math Plugin

The {productname} {release-version} release includes an accompanying release of the **Math** premium plugin.

This **Math** premium plugin release includes the following fix:

> ==== Math icon state was not set to active on math element selection.

Author: ```suggestion
==== Math icon state was not set to active on math element selection.
// #TINY-10931
```

The following premium plugin updates were released alongside {productname} {release-version}.

=== Math Plugin

The {productname} {release-version} release includes an accompanying release of the **Math** premium plugin.

This **Math** premium plugin release includes the following fix:

==== Math icon state was not set to active on math element selection.
// #TINY-10931

Previously, the Math icon was not set to active when a math element was selected within the editor.

As a consequence, users had no visual indication that a math element was selected or that it could be edited by selecting it and then clicking the toolbar button.

> {productname} {release-version} addresses this issue. Now, when a tiny-math-inline or tiny-math-block HTML element is selected, the button's active state is triggered.

Reviewer: ```suggestion
{productname} {release-version} addresses this issue. Now, when any Math element is selected, the button's active state is triggered.
```
